ProfileManager.js

```diff
@@ -95,14 +95,26 @@ export default class ProfileManager {
 
   async createUserDocument({
     edvClient, invocationSigner, profileAgentId, referenceId,
-    content = {zcaps: {}}
+    content = {}
   }) {
     edvClient.ensureIndex({attribute: 'content.profileAgentId'});
 
+    // FIXME: content.id *must* be the profileId if this is by design
+    // how can the ID best be validated?
+    // content.id is used to dereference profileAgents
+    if(!content.id) {
+      throw new TypeError('"content.id" is required.');
+    }
+
     // create the user document for the profile agent
     const userDocument = await edvClient.insert({
       doc: {
-        content: {...content, profileAgentId},
+        content: {
+          ...content,
+          profileAgentId,
+          type: ['Person', 'User'],
```

Comment:
Type is not always `['User', 'Person']`. For example, it could be `['User', 'Application']`.

Reply:
OK, should this not be defaulted then and `content` should always include a `type` as with `id`?

Reply:
Yes, we should include `type` in content.